boards/shields/esp_8266/esp_8266.overlay

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright (c) 2020 Gerson Fernando Budke <nandojve@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* File meant to be empty */
```

Comment:
Can you add a comment stating why this generic overlay can't be used  ?

Reply:
Done!.

I was thinking about this issue and I was wondering if besides _shield_foo.overlay_ an _arduino.overlay_ could not be useful inside shields directory. I don't know how complex will be evaluate the "feature" arduino on DT and then include _arduino.overlay_. With that, I think, we can solve this kind of issues.